src/components/panels/HistoryListPanel.vue

```diff
@@ -14,15 +14,19 @@
                         dense />
                 </v-col>
                 <v-col class="offset-4 col-4 d-flex align-center justify-end">
-                    <template v-if="selectedJobs.length">
-                        <v-btn
-                            :title="$t('History.Delete')"
-                            color="error"
-                            class="px-2 minwidth-0 ml-3"
-                            @click="deleteSelectedDialog = true">
-                            <v-icon>{{ mdiDelete }}</v-icon>
-                        </v-btn>
-                    </template>
+                    <v-tooltip top>
+                        <template #activator="{ on, attrs }" v-if="selectedJobs.length">
```

Comment:
i would move the v-if from the template to the v-tooltip, because the empty tooltip is not needed, when the button is also hidden.